### PR TITLE
Now that luarocks is whitelisted, uncomment full matrix.

### PR DESCRIPTION
**Pull request by tylercloke**
We had a bunch of stuff commented out since luarocks was not whitelisted,
but it was via SSD's ticket:

https://github.com/travis-ci/travis-ci/issues/3716

### CODE REVIEW

File: src/bookshelf/Makefile [hunk 1]
```diff
@@ -25,6 +25,9 @@ all: $(REBAR3)
 rel: all
 	@$(REBAR3) release
 
+update: $(REBAR3)
```
Comment (marcparadise) [inline src/bookshelf/Makefile]: Were these  pulled in from concrete upstream?   If not, it'll be overwritten on the next concrete update (same for the other Makefile changes)

**Comment by marcparadise** [comment]
Other than confirming the origination of the Makefile changes, LGTM

File: src/bookshelf/Makefile (hunk 1 above)
Comment (tylercloke) [inline src/bookshelf/Makefile]: Nope, no idea how new tooling works.

Comment (joedevivo) [inline src/bookshelf/Makefile]: CONCRETE IS DEAD! LONG LIVE CONCRETE!

**Comment by marcparadise** [comment]
I've since learned that we have removed concrete from our pipeline, merge when ready :+1:

**Pull request merged**